templates/test/ci/prow-user-assigned-managed-identity/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
why is this file being added? seems unrelated, maybe a rebase conflict?

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2456#discussion_r953604128